playbook.yaml

```diff
@@ -26,14 +26,15 @@
   roles:
     - { role: validate-input, tags: ['validate-input'] }
     - { role: validate-input-apigee-control-plane, tags: ['validate-input', 'validate-input-apigee'] }
-    - { role: validate-input-k8s-cluster, tags: ['validate-input', 'validate-input-k8s'] }
+    - { role: validate-input-k8s-cluster, vars: { custom_cassandra_storage_class: '{{ overrides.cassandra.storage.storageClass }}' }, tags: ['validate-input', 'validate-input-k8s'] }
```

Comment:
@RiflerRick : No need to explicitly send the variable as it is accessible in the python scripts 

As you can do something like below in the task

```
- name: execute validate kubernetes platform script
  block:
    - script: |
        validate_k8s_objects.py \
        --input_data '{{ overrides_string }}' \
        --generate_certificates '{{ generate_certificates }}' \
        --create_service_account '{{ create_service_account }}' \
        --cassandra_storage_class '{{ overrides.cassandra.storage.storageClass }}'
   ```

Reply:
sure